settings.py

```diff
@@ -1,12 +1,15 @@
 import sys
 import os
 import subprocess
+from glob import glob
+import shutil
 
 import bpy
 from bpy.types import AddonPreferences
 from bpy.props import BoolProperty, FloatVectorProperty, EnumProperty, IntProperty, FloatProperty, StringProperty
 
 from sverchok.ui.utils import message_on_layout
+from sverchok.ui.presets import get_sverchok_directory
 
 """Don't import other Sverchok modules here"""
```

Comment:
You are going to initialize half of Sverchok architecture via the settings module again.

Reply:
okay...
get_sverchok_directory is quite simple function, it should be somewhere in core I think...